src/app/shared/dockstore.model.ts

```diff
@@ -104,5 +104,6 @@ export class Dockstore {
     enableLaunchWithGalaxy: true,
     enableMultiCloudLaunchWithDNAstack: false,
     enableOrcidExport: true,
+    enableSnakemake: false,
```

Comment:
think we can turn this on for a staging release

Reply:
Hmm this value is controlled by the presence of `?snakemake` in the URL. Are you saying we should turn it on for staging regardless of the `?snakemake` parameter in the URL?

Reply:
> I don't see the purpose in having the URL "turn off" the feature temporarily if the default is on.

Yeah, you're probably right. I was probably more interested in being able to turn on/off the feature globally. Agreed, the url control probably only be useful if if the default was off. 

> In the future, it might be better to control the features from the webservice through the web.yaml so that we don't need a new UI release to turn a feature on/off 

I can see the argument, config change vs new build is more convenient. Feels a little strange to flip a change in the UI from the webservice, but I don't see a specific problem.

Reply:
I'll set the snakemake default as true so it's on and remove the URL control